research/using-npm-link.md

```diff
@@ -0,0 +1,37 @@
+Получилось найти [одно решение](https://github.com/facebook/react/issues/13991#issuecomment-1867954003) где предлагалось в node\_modules прилинкованного пакета удалить React и линкануть его из node\_modules сервиса, в который подключили пакет (5 шаг в Solution).  
```

Comment:
Let's add some introduction into our problem that we were solving? What we needed to do in the first place, in which repo, what we tried to achieve with npm link, what went wrong. Otherwise, it isn't clear for others where we started.